### PR TITLE
Footer Popup extends PopupProps

### PR DESCRIPTION
**Pull request by JKirchartz**
FooterPopupProps should extent PopupProps instead of CommonProps to allow access to all internal props.

### CODE REVIEW

**Review by bsteinbk — APPROVED** [review]

**Review by pmconne — APPROVED** [review]
Not sure why separate approval needed for changelogs.

**Comment by JKirchartz** [comment]
> Not sure why separate approval needed for changelogs.

I made the initial edit on GitHub yesterday afternoon, but it doesn't appear that the pre-commit hooks were causing the `rush change -v` (verification step) to fail because I never ran `rush change`. I cloned the project and generated the change on a different commit and pushed that up just now.

File: ui/ninezone/src/ui-ninezone/footer/Popup.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export enum FooterPopupContentType {
 /** Properties of [[FooterPopup]] component.
  * @beta
  */
-export interface FooterPopupProps extends CommonProps {
+export interface FooterPopupProps extends PopupProps {
```
[the remark this answers is not in the document]
Comment (JKirchartz) [reply ui/ninezone/src/ui-ninezone/footer/Popup.tsx:28]: I would like to use set [closeOnEnter from Popup](https://github.com/imodeljs/imodeljs/blob/2f380c48cf8351fba41aad73c3ab682dd324e51d/ui/core/src/ui-core/popup/Popup.tsx#L66) on the [Popup element in this module](https://github.com/imodeljs/imodeljs/blob/master/ui/ninezone/src/ui-ninezone/footer/Popup.tsx#L70).

Can I use a regular Popup element in place of a FooterPopup in the footer? Maybe I'm using the wrong element.

[the remark this answers is not in the document]
Comment (GerardasB) [reply ui/ninezone/src/ui-ninezone/footer/Popup.tsx:28]: You could, but you would loose some of the styling and configuration. 
I was a bit worried about allowing to overwrite preconfigured props, but that should not be a problem. Let's go ahead and merge this.

**Review by GerardasB — APPROVED** [review]

**Review by calebmshafer — APPROVED** [review]

**Comment by calebmshafer** [comment]
@JKirchartz can you take a look at the build failures?

**Comment by JKirchartz** [comment]
@calebmshafer I can't work past the build failures, I only really need the `closeOnEnter` prop added.

**Comment by calebmshafer** [comment]
@JKirchartz the build is failing due to API extractor.  Can you please do a clean build and then run `npm run extract-api` in the ui-ninezone package?

**Review by calebmshafer — CHANGES_REQUESTED** [review]
Adding this is an actual review instead of just a comment.  Would you be able to run the extract-api command to update those files?  This is blocking the CI build from passing.

**Comment by GerardasB** [comment]
Closing in favor of #385

**Pull request closed without merging**